AUTHORS

```diff
@@ -1128,3 +1128,8 @@ Leo Battle <leowbattle@gmail.com>
 Advait <apote2050@gmail.com>
 Anurag Bhat <bhat.1@iitj.ac.in>
 Jeremy Monat <jemonat@calalum.org>
+NikhilSDate <47920034+NikhilSDate@users.noreply.github.com>
+Diane Tchuindjo <dtchuindjo@gmail.com>
+Tom Fryers <61272761+TomFryers@users.noreply.github.com>
+Zouhair <zouhair.mahboubi@gmail.com>
+Yurii Kukharchuk <ykuharchuk@gmail.com>
```

Comment:
These changes to the AUTHORS file need to be reverted. Only .mailmap should be changed.

Reply:
Reverted.